### ISSUE TITLE
Please use GitHub's "release" and/or "pre-release" tags

### ISSUE THREAD

**Issue by seemantk** [randombit/botan#2940]
Hello from Funtoo Linux!  Over here, we are automating our package updates by looking at upstream github repos.   With the current method, the latest tag that comes up is 3.0.0-alpha1, rather than 2.19.1 (which is the actual latest stable).

Our request is to please use the tagging mechanism that GitHub provides to make the tag/releases easier to parse by machine.

Thank you so much for your consideration of this issue

**Comment by randombit** [comment]
Shouldn't for this you just filter out any tags with `alpha`, `beta`, `rc`, etc?

**Comment by seemantk** [comment]
> Shouldn't for this you just filter out any tags with `alpha`, `beta`, `rc`, etc?

Of course.  It would also be good information management practice to use the tagging system that the GitHub platform provides which makes things much clearer for downstreams (e.g. linux distros) looking to package this.

**Comment by randombit** [comment]
Can you give some background on the difference between tags in Github and the normal `git` tags that we currently use? I don't really see what additional information is given by the GH tags. Maybe I'm missing something here, unfortunately https://docs.github.com/ seems to be down/unavailable at the moment so search isn't helping clear this up.

**Comment by seemantk** [comment]
Thank you, this is the doc that I'm able to find: https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository
